### PR TITLE
Upgrade lib-postgresql, org.webjars.webjars-play

### PR DESCRIPTION
**Pull request by mikeclimbrock**
- io.flow
    - lib-postgresql (0.0.58 => 0.0.60)
  - org.webjars
    - webjars-play (2.6.2 => 2.6.3)

### CODE REVIEW

**Comment by mikeclimbrock** [comment]
updated routes file for `webjars` per:
https://github.com/webjars/webjars/commit/72885ef8b737a70abb5e86976a6f464fcfa6ce40#diff-665b5e98133dea78e2a31206d75f555bR81

File: www/conf/routes [hunk 1]
```diff
@@ -73,4 +73,4 @@ POST    /:org/members/postMakeAdmin             controllers.MembersController.po
 
 # Map static resources from the /public folder to the /assets URL path
 GET     /assets/*file               controllers.Assets.at(path="/public", file)
-GET     /webjars/*file              controllers.WebJarAssets.at(file)
\ No newline at end of file
+GET     /webjars	            webjars.Routes
```
Comment (benwaffle) [inline www/conf/routes]: oh no you used tabs instead of spaces

Comment (mikeclimbrock) [reply www/conf/routes]: updated formatting

**Comment by mikeclimbrock** [comment]
updated routes file per sample provided in documentation:
https://github.com/webjars/webjars-play/blob/master/src/main/resources/webjars.routes

**Pull request merged**